### PR TITLE
Typo in comment fixed

### PR DESCRIPTION
**Pull request by King2500**
I think this is probably a typo and was meant to be "semver" (for semantic versioning).

Right?

### CODE REVIEW

**Comment by King2500** [comment]
Ping @javiereguiluz

File: src/Symfony/Installer/NewCommand.php [hunk 1]
```diff
@@ -137,7 +137,7 @@ private function checkSymfonyVersionIsInstallable()
             return $this;
         }
 
-        // validate server syntax
+        // validate semver syntax
```
Comment (xabbuh) [inline src/Symfony/Installer/NewCommand.php]: I would actually change this to "validate version syntax" or "validate version pattern".

**Comment by javiereguiluz** [comment]
Thank you @King2500.

File: src/Symfony/Installer/NewCommand.php (hunk 1 above)
Comment (javiereguiluz) [inline src/Symfony/Installer/NewCommand.php]: @xabbuh your proposal is easier to understand ... but any developer looking at the source code of this command probably knows what `semver` is :)

**Pull request merged**